src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/NodeRunners.cs

```diff
@@ -93,23 +94,25 @@ public static FullNode BuildFullNode(NodeSettings args, Action<IFullNodeBuilder>
             }
             else
             {
-                node = (FullNode)new FullNodeBuilder()
+                var builder = new FullNodeBuilder()
                     .UseNodeSettings(args)
                     .UsePosConsensus()
                     .UseBlockStore()
                     .UseMempool()
                     .UseWallet()
                     .AddPowPosMining()
                     .AddRPC()
-                    .MockIBD()
-                    .Build();
+                    .MockIBD();
+
+                if (this.mineCoinsFast)
```

Comment:
Create a new runner and remove this flag `mineCoinsFast`

Reply:
Fixed!